presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -192,6 +192,14 @@ public BaseJdbcClient(
         }
     }
 
+    protected boolean filterSchema(String schemaName)
```

Comment:
`isUserSchema`? (as opposed to internal schema)

Reply:
in line with `io.prestosql.plugin.hive.HiveMetadata#filterSchema`